src/lsptoolshost/commands.ts

```diff
@@ -178,7 +178,7 @@ async function openSolution(languageServer: RoslynLanguageServer): Promise<vscod
         return undefined;
     }
 
-    const solutionFiles = await vscode.workspace.findFiles('**/*.sln');
+    const solutionFiles = await vscode.workspace.findFiles('**/*.{sln,slnf}');
```

Comment:
Does the server side actually work for this?  Specifically I don't know if the msbuild API we're calling parses the sln file
https://github.com/dotnet/roslyn/blob/main/src/Workspaces/Core/MSBuild.BuildHost/BuildHost.cs#L159

Reply:
This does work. Verified with local testing and reviewing the MSBuild API itself. Looks like we could simplify some MSBuildWorkspace handling of Solution Filters by using this same api.